src/PIL/ImageFont.py

```diff
@@ -230,7 +230,7 @@ def load_from_bytes(f):
             )
 
         if is_path(font):
-            if isinstance(font, Path):
+            if isinstance(font, os.PathLike):
                 font = str(font)
```

Comment:
For `pathlib.Path`s, `str(path_object)` always produces what you want here, but that's not necessarily true for all `os.PathLike`s. You should use `os.fspath` here if you want the code to work with arbitrary `PathLike`s:

```suggestion
                font = os.fspath(font)
```

You can actually also pass `str` and `bytes` objects to `os.fspath`, so you could probably get rid of the `isinstance(font, os.PathLike)` check immediately above as well (I'm on my mobile so my suggestion can't span multiple lines :( )

Reply:
Docs: https://docs.python.org/3/library/os.html#os.fspath